src/team_lib/mhba/xccl_mhba_collective.c

```diff
@@ -537,25 +750,13 @@ xccl_status_t xccl_mhba_alltoall_init(xccl_coll_op_args_t  *coll_args,
     block_size = team->requested_block_size ? team->requested_block_size : block_size;
 
     //todo following section correct assuming homogenous PPN across all nodes
```

Comment:
comment no longer relevant?

Reply:
relevant. we still do not support non-homogenous PPN, at all.